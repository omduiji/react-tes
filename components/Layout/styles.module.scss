.layout {
    display: flex;
    justify-content: center;
    flex-direction: column;
    min-height: 100vh;

    @media (min-width: 500px) {
        flex-direction: row;
        align-items: center;
    }
}
