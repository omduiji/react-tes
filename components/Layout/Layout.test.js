import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import Layout from '.'

test('should render Layout', () => {
  const { queryByText, asFragment } = render(<Layout><p>test content</p></Layout>)

  expect(queryByText('test content')).toBeInTheDocument()

  expect(asFragment()).toMatchSnapshot()
})
