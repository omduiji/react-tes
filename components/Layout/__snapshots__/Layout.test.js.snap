// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Layout 1`] = `
<DocumentFragment>
  <div
    class="layout"
  >
    <p>
      test content
    </p>
  </div>
</DocumentFragment>
`;
